pkg/yang/entry.go

```diff
@@ -1485,6 +1485,13 @@ func (e *Entry) merge(prefix *Value, namespace *Value, oe *Entry) {
 		} else {
 			v.Parent = e
 			v.Exts = append(v.Exts, oe.Exts...)
+			for lk, lv := range oe.Extra {
+				if v.Extra[lk] == nil {
+					v.Extra[lk] = lv
+					continue
+				}
+				v.Extra[lk] = append(v.Extra[lk], oe.Extra[lk]...)
```

Comment:
(optional) Note that currently only augments in `uses` statements are going to be picked up. Raw augments won't be. I'd also suggest adding this statement outside of the for loop as well and fixing the relevant test.

Reply:
Can you please give an example where this would occur?

Reply:
For what I see, `augment` defined in `uses` are ignored, regardless of if-feature field - could we open a new issue, approve this PR, and treat that corner case separately?  What do you reckon @wenovus ?